blog article for tivity health use case (#13063)
pulumi/docs: content/blog/devsecops-strategy-security-automation-tivity-health/index.md

---

> title: "Security Automation for Faster, Cheaper, and Better Results"

Author: ```suggestion
title: "DevSecOps Game-Changer: Security Automation That Delivers Real Business Results"
```

> title: "Accelerating Security Automation for Faster, Cheaper, and Better"

Author: ```suggestion
title: "Accelerating Security Automation for Faster, Cheaper, and Better Results"
```

> title: "Accelerating Security Automation for Faster, Cheaper, and Better Results"

Author: ```suggestion
title: "Security Automation for Faster, Cheaper, and Better Results"
```

> title: "DevSecOps Game-Changer: Security Automation That Delivers Real Business Results"

Author: ```suggestion
title: "DevSecOps Game-Changer: Security Automation That Delivers Business Results"
```

allow_long_title: true

# The date represents the post's publish date, and by default corresponds with
# the date and time this file was generated. Dates are used for display and
# ordering purposes only; they have no effect on whether or when a post is
# published. To influence the ordering of posts published on the same date, use
# the time portion of the date value; posts are sorted in descending order by
# date/time.

> date: 2024-10-10T07:16:30Z

Author: ```suggestion
date: 2024-11-21T07:16:30Z
```

# The draft setting determines whether a post is published. Set it to true if
# you want to be able to merge the post without publishing it.
draft: false

# Use the meta_desc property to provide a brief summary (one or two sentences)
# of the content of the post, which is useful for targeting search results or
# social-media previews. This field is required or the build will fail the
# linter test. Max length is 160 characters.
meta_desc: Accelerate security automation to achieve faster, cheaper, and more secure cloud deployments. Discover Tivity Health's cloud transformation journey with Pulumi

# The meta_image appears in social-media previews and on the blog home page. A
# placeholder image representing the recommended format, dimensions and aspect
# ratio has been provided for you.

> meta_image: meta.png

Author: ```suggestion
meta_image: david-tivity-healthy-quote.png
```

authors:
    - sara-huddleston

# At least one tag is required. Lowercase, hyphen-delimited is recommended.
tags:
    - security
    - devecops
    - devops
    - infrastructure-as-code
    - platform-engineering
    - infrastructure-lifecycle-management
    - developer-experience-devex

# The social copy used to promote this post on Twitter and Linkedin. These
# properties do not actually create the post and have no effect on the
# generated blog page. They are here strictly for reference.

# Here are some examples of posts we have made in the past for inspiration:
# https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
# https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
# https://twitter.com/PulumiCorp/status/1763265391042654623
# https://twitter.com/PulumiCorp/status/1762900472489185492
# https://twitter.com/PulumiCorp/status/1755637618631405655

social:
    twitter:
    linkedin:

# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md
# for details, and please remove these comments before submitting for review.
---

> In today's cloud-driven business landscape, organizations are pressured to deliver new products and features faster than ever. However, they must also balance rapid deployment, security, and cost-effective infrastructure management. 

Author: ```suggestion
Organizations are under constant pressure to deliver new products and features faster than ever. But speed alone isn’t enough—businesses must also navigate the complex challenges of ensuring security and managing infrastructure costs effectively.
```

Tivity Health, a leading health and fitness solutions provider, has embraced this DevSecOps approach using Pulumi, a modern infrastructure as code (IaC) platform. During PulumiUP 2024, David Giambruno, Tivity Health's VP of Engineering and DevOps, shared how, by leveraging Pulumi, he led the transformation that continuously drives remarkable results in speed, cost savings, and security.

<!--more-->

## On this article:

- [The Beginning: From Data Center to the Cloud](/blog/devsecops-strategy-security-automation-tivity-health/#the-beginning-from-data-center-to-the-cloud)
- [Embracing Pulumi: The Power of Automation, Productivity, and Security](/blog/devsecops-strategy-security-automation-tivity-health/#embracing-pulumi-the-power-of-automation-productivity-and-security)
- [Driving Business Impact Through Security Automation](/blog/devsecops-strategy-security-automation-tivity-health/#driving-business-impact-through-security-automation)

> - [Fostering DevOps Culture Through Cross-Functional Collaboration](/blog/devsecops-strategy-security-automation-tivity-health/fostering-devops-culture-through-cross-functional-collaboration)

Author: ```suggestion
- [Fostering DevOps Culture Through Cross-Functional Collaboration](/blog/devsecops-strategy-security-automation-tivity-health/#fostering-devops-culture-through-cross-functional-collaboration)
```

> - [Fostering DevOps Culture Through Cross-Functional Collaboration](/blog/devsecops-strategy-security-automation-tivity-health/fostering-devops-culture-through-cross--functional-collaboration)

Author: ```suggestion
- [Fostering DevOps Culture Through Cross-Functional Collaboration](/blog/devsecops-strategy-security-automation-tivity-health/fostering-devops-culture-through-cross-functional-collaboration)
```
